src/components/dashboard/HandlePaymentLink.js

```diff
@@ -190,7 +190,7 @@ const HandlePaymentLink = (props: HandlePaymentLinkProps) => {
               category: ExceptionCategory.Human,
               dialogShown: true,
             })
-            await showErrorDialog(withdrawnOrSendError, undefined, { onDismiss: screenProps.goToRoot })
+            showErrorDialog(withdrawnOrSendError, undefined, { onDismiss: screenProps.goToRoot })
```

Comment:
we need await here, also in any structure like

```
try {
// ...
showDialog()
// ...
} catch {
  showErrorDialog()
}
```

all show dialog calls in `try` block should be `await`ed

Reply:
@johnsmith-gooddollar @sirpy requested to revert everything back to before we implemented, and only apply it for what we wanted to fix.

And as far as I know the only place were I know it was broken, was the taskDialog (which was kept)

Reply:
re check everything please. you you 10000% sure, you could merge